pandas/tests/io/excel/test_readers.py

```diff
@@ -166,13 +167,19 @@ def test_usecols_int(self, read_ext):
                 usecols=3,
             )
 
-    def test_usecols_list(self, request, read_ext, df_ref):
-        if read_ext == ".xlsb":
+    def test_usecols_list(self, request, engine, read_ext, df_ref):
+        if engine == "pyxlsb":
             request.node.add_marker(
                 pytest.mark.xfail(
                     reason="Sheets containing datetimes not supported by pyxlsb"
                 )
             )
+        if engine == "calamine" and read_ext in {".xls", ".xlsb", ".ods"}:
+            request.node.add_marker(
+                pytest.mark.xfail(
+                    reason="Calamine support parsing datetime only in xlsx"
```

Comment:
How realistic is it to support this in the other formats? I'm a little wary of making our reader-landscape more complicated

Reply:
I have made pr with detecting datetime for ods, for example (https://github.com/tafia/calamine/pull/288). But, 1) I'm not maintainer of rust library 2) making calamine more compatible with python libraries will take a long time, but it's possible.

Reply:
Will merge your changes.

Reply:
It looks like the PR linked above was merged - should ods be removed from this xfail?